docs/Researcher/user-interface/workspaces/create/create-ds.md

```diff
@@ -19,27 +19,27 @@ The data can be set as read-write or limited to read-only permission regardless
 To create an PVC data source, provide:
 
 * A data source name
-* A Run:ai project scope
+* A Run:ai project scope which is assigned to that item and all its subsidiaries.
 * Select an existing PVC or create a new one by providing:
 
-    * a claim name
-    * a storage class
-    * access mode
-    * required storage size
-    * volume system mode
+    * A claim name
+    * A storage class
+    * Access mode
+    * Required storage size
+    * Volume system mode
 
```

Comment:
```suggestion
Note: If you selected an existing PVC & a department scope, you must add the following label on the secret in the cluster 'run.ai/department: <department-id>'. This will enable the automatic replication of this secret to all the relevant subsidiaries.
```